packages/@react-stately/color/src/Color.ts

```diff
@@ -304,6 +307,8 @@ class HSBColor extends Color {
         return `hsb(${this.hue}, ${toFixedNumber(this.saturation, 2)}%, ${toFixedNumber(this.brightness, 2)}%)`;
       case 'hsba':
         return `hsba(${this.hue}, ${toFixedNumber(this.saturation, 2)}%, ${toFixedNumber(this.brightness, 2)}%, ${this.alpha})`;
+      case undefined:
```

Comment:
need to account for this technically illegal use of toString because some projects, storybook for example, call `String` on all the props it can see in a story, leading to `toString` being called with `undefined`

Reply:
Is this the same reason for above 'undefined' on line 122?